data/aux/sdg_index_mappings.json

```diff
@@ -0,0 +1 @@
+{"trend_map": {".": NaN, "\u2191": "increasing_on_track", "\u279a": "increasing_moderately", "\u2192": "stagnant", "\u2193": "decreasing"}, "achievement_map": {"green": "goal_achievement", "yellow": "challenges_remain", "orange": "significant_challenges", "red": "major_challenges"}}
```

Comment:
Does the NaN value get read as valid json? 

You can also set the `na_values` param in `pandas.read_excel` https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html

Reply:
Remove `".": NaN` since it represents a missing value and is automatically mapped to `NaN` when mapping with pandas.